src/App.css

```diff
@@ -782,6 +782,24 @@ body.dark {
   z-index: 11; /* above breadcrumbs, toolbar, export modal and status */
 }
 
+.alert-inline {
+  position: absolute;
+  top: 45px;
+  padding: 10px 0;
+  z-index: 11;
+  color: gray;
+  width: 100%;
+  text-align: center;
+  font-size: small;
+}
+
+.alert-inline-text {
+  padding: 4px 10px;
+  border-radius: 8px;
+  border: 1px solid gray;
+  text-transform: capitalize;
+}
```

Comment:
Can we try this with exactly the same styles (position, color, size, etc) as the existing alert, just inline instead of full width? We can add other styles if it needs to stand out more, but I'd like to try the requested change only first.

Reply:
Replicating the same style will not work as intended for small size screens as it will still block some of the toolbar shortcuts. Do you want me to leave as it is for now?

Reply:
Hmmm good point. I'll have to think about that. I don't like how much the new style deviates from the old style, but I can't think of an alternative right now. We can leave this as-is and I can think of some ideas after I have used it in production for a bit. Thanks!